css/styles.css

```diff
@@ -0,0 +1,157 @@
+* {
+	background-color: #FFFFFF;
+	font-family: sans-serif;
+}
+
+body {
+	width: 800px;
+	margin: auto;
+}
+
+nav {
```

Comment:
:hammer: I notice that in your HTML you're using the `<nav>` elements as a way to create horizontal lines. While this works, it wouldn't be best practice to use these _semantic tags_ for that purpose. It would be preferred here to use a `<div>` instead. But, as you see, you can pretty much do whatever you want with whatever element you want, so long as the css is right. Just pointing out that the `<nav>` tags probably weren't intended to be used in this manner.